### DISCUSSION TITLE
about

### DISCUSSION THREAD

**Discussion by giscus** [STranslate/STranslate#71] [Announcements]
# about

WPF 开发的一款即开即用、即用即走的翻译(OCR)工具

https://stranslate.zggsong.com/about.html

**Comment by elfcute** [comment]
恳求添加“谷歌OCR"服务，让用户自行添加api 
谷歌ocr的识别超强
求大佬添加一个谷歌ocr服务，这样用户就可以自行输入谷歌ocr的apikey来配置了，球球了

**Comment by ZGGSONG** [comment]
ok

**Comment by jasonet** [comment]
Mac OS

**Reply by ZGGSONG** [reply]
wpf为windows ui框架

**Comment by batemax** [comment]
请问什么时候可以支持Azure OpenAI服务？Azure的格式与普通的OpenAI不太一样

**Reply by ZGGSONG** [reply]
最新版已经支持了呀

**Reply by batemax** [reply]
抱歉，没注意Github已经更新了，官网链接版本号还是1.2.2.904

**Reply by ZGGSONG** [reply]
最近比较忙，文档仓库忘了更新

**Comment by MonkeyKiller2330** [comment]
最新版本，本地STranslate翻译连通测试显示失败

**Reply by ZGGSONG** [reply]
> 最新版本，本地STranslate翻译连通测试显示失败

大概率网络问题，可以挂代理试试

**Comment by guoyuyuan** [comment]
下载的orc服务文件，安装在哪个路径下

**Reply by ZGGSONG** [reply]
PaddleOCR数据包下载方式：
1. 软件内直接下载（github，ghproxy网络不好的人不推荐）
2. Release页面中网盘页面下载最新的PaddleOCR数据包丢到软件根目录后点击下载即可(方式任意，会首先检查本地是否存在)

> 软件支持使用WeChatOCR，速度更快，效果更好

**Comment by MerlinAmbr0sius** [comment]
1.2.10.1117版本直接打不开了

**Reply by ZGGSONG** [reply]
日志看下呢，issue反馈吧，谢谢

**Comment by yizhao266** [comment]
可以添加下面几个特性吗？
1.命令行图像导入：可以通过命令行导入图像进行 OCR 操作。
2.命令行pdf&txt文件/文本导入：可以通过命令行导入pdf&txt文件/文本进行 OCR 操作，如果可支持命令配置翻译引擎更好。
3命令行剪贴板文本翻译：可以通过命令行在启动 STranslate 时进行 翻译 操作，如果可支持命令配置翻译引擎更好。
4.启动程序执行截图 OCR：在启动 STranslate 时直接进行截图 OCR 操作
5.微信路径设置支持相对路径。

**Reply by ZGGSONG** [reply]
1-3. https://stranslate.zggsong.com/docs/invoke.html#api%E8%B0%83%E7%94%A8
4. 当前无计划
5. 当前无计划,难道你要天天该微信安装目录啊

> 特殊需求: https://github.com/ZGGSONG/STranslate?tab=readme-ov-file#others

**Comment by Yingxi04520** [comment]
能不能添加一个屏蔽窗口功能来替换掉(全屏模式下忽略热键)这个功能 我在有的游戏上（窗口化全屏）想使用翻译功能 但是打开了这个功能后就无法呼出了 但在像cs2这种游戏上（非窗口化全屏）我希望禁用（因为呼出来后会退回桌面） 就导致我玩不同游戏是还是需要来回的开关这个功能 比较的麻烦

**Reply by ZGGSONG** [reply]
有全屏模式忽略全局热键的功能啊，常规设置-常用配置-全屏模式下忽略热键

**Comment by Yingxi04520** [comment]
我是希望能够在一部分全屏应用下使用该功能而不用重新调出面板进行切换




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ZGGSONG/STranslate"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年12月14日(星期六) 晚上7:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ZGGSONG/STranslate] about (Discussion #71)





 
有全屏模式忽略全局热键的功能啊，常规设置-常用配置-全屏模式下忽略热键

**Reply by ZGGSONG** [reply]
提feature吧 有空的时候我会找feature的功能去实现
